deploy/charts/rook-ceph-cluster/values.yaml

```
@@ -55,6 +55,17 @@ monitoring:
 # If true, create & use PSP resources. Set this to the same value as the rook-ceph chart.
 pspEnable: true
 
+extended-ceph-exporter:
```

```suggestion
extendedCephExporter:
```

Let's also add a short concise comment here and link to the extended-ceph-exporter Chart README.md

---

yeah that would be good! good catch, thanks Alex!